my-app/pages/navbar/index.tsx

```diff
@@ -0,0 +1,25 @@
+import logo from "../../public/assets/images/wecare_logo.svg";
+import styles from "./navbar.module.scss";
+import Image from "next/image";
+
+const Navbar = () => {
+  return (
+    <>
+      <nav className={styles.main}>
+        <div className={styles.logo}>
```

Comment:
Why is this class is here? I'm not seeing any class in the respective scss file.